client/components/repository/Settings/UserManagement/AddUserDialog.vue

```diff
@@ -74,10 +84,8 @@ export default {
   },
   methods: {
     ...mapActions('repository', ['upsertUser']),
-    addUser() {
+    submit() {
       setTimeout(async () => {
-        const isValid = await this.$validator.validateAll();
```

Comment:
If you just type in email of the existing user and click `add` validation will fail